symengine/series_flint.h

```diff
@@ -10,12 +10,13 @@
 namespace SymEngine
 {
 
-using fp_t = fmpq_poly_wrapper;
+using fqp_t = fmpq_poly_wrapper;
```

Comment:
In two places there was
`using fp_t = fmpq_poly_wrapper`
`using fp_t = fmpz_poly_wrapper`

It was causing issues, as I needed to use the `fmpz` version but the `fmpq` one was defined later (in the includes). So changed it to end the ambiguity.